service_user.go

```diff
@@ -78,6 +85,24 @@ func (h *ServiceUsersHandler) Get(project, serviceName, username string) (*Servi
 	return nil, err
 }
 
+// Put modifies the given Service User in Aiven.
+func (h *ServiceUsersHandler) Put(project, service, username string, password string) (*ServiceUser, error) {
+	msr := ModifyServiceUserRequest{Operation: "reset-credentials", NewPassword: password, Authentication: "caching_sha2_password"}
+	path := buildPath("project", project, "service", service, "user", username)
+	svc, err := h.client.doPutRequest(path, msr)
+	if err != nil {
+		return nil, err
+	}
+	var r ServiceResponse
+	errR := checkAPIResponse(svc, &r)
+	for _, user := range r.Service.Users {
+		if user.Username == username {
+			return user, nil
+		}
+	}
+	return nil, errR
```

Comment:
There is a possibility to get `nil, nil` which is not conventional for functions with such signature, would be nice to get some error (at least something like `errors.New("not found")`) if the first argument is `nil`, so that users of this function will get something for the first argument if `error` is `nil`.